core/src/main/java/com/expedia/adaptivealerting/core/metric/MPoint.java

```diff
@@ -0,0 +1,42 @@
+package com.expedia.adaptivealerting.core.metric;
+
+import com.expedia.www.haystack.commons.entities.MetricPoint;
+
+public class MPoint {
```

Comment:
There's already an Mpoint class. (If you want to change it to MPoint that's OK with me.)